db/seeds/activities/preprod_achiever_courses.rb

```diff
@@ -0,0 +1,48 @@
+# These courses exist on the preprod instance of Dynamics for testing: so they are visible on
+# the preprod Smart Connector (Achiever)
+
+cs_accelerator = Programme.cs_accelerator
+primary_certificate = Programme.primary_certificate
+secondary_certificate = Programme.secondary_certificate
+
+########################################################################################################################
+
+a = Activity.find_or_create_by(stem_course_template_no: 'f5fe2274-fdb1-ed11-83ff-0022481b547a') do |activity|
+  activity.title = 'NCCE Online Course 1'
+  activity.credit = 20
+  activity.slug = 'ncce-online-course-1'
+  activity.category = 'online'
+  activity.provider = 'stem-learning'
+  activity.stem_activity_code = 'OC123'
+  activity.always_on = true
```

Comment:
will you still be leaning on the `always_on` behaviour?

Reply:
Yes, for now. I think I deleted one place where `always_on` is used, but otherwise things break if that flag is nil or false.

We should make a ticket for a future sprint to retire that flag.